File: Day1-10/Day5/NextRound.cpp
#include <iostream>
using namespace std;
int main() {
    int size,position;
    scanf("%d %d",&size,&position);
    int arr[size],nextRound;
    for(int i=0;i<size;i++){
        scanf("%d",&arr[i]);
    }
    for(int i=0;i<size;i++){
        if(arr[i]>=arr[position-1]&&arr[i]>0){
            nextRound++;
        }
    }
    printf("%d",nextRound);
    return 0;
}

File: Day1-10(800)/Day5/NextRound.py
# this is giving run time error becuse it is slow in python we can do this in cpp it is working 
size, position = map(int, input().split())
arr=[]
nextRound=0
for i in range(size):
    x=int(input())
    arr.append(x)
for i in range(size):
    if arr[i]>=arr[position-1] and arr[i]>0:
        nextRound+=1
print(nextRound)
